packages/runtime-vapor/src/component.ts

```diff
@@ -0,0 +1,34 @@
+import { EffectScope } from '@vue/reactivity'
+import { Block, BlockFn } from './render'
+
+export interface ComponentInternalInstance {
+  uid: number
+  container: ParentNode
+  block: Block
```

Comment:
Should it be nullable? It could be accessed by setup function.

Reply:
fixed! https://github.com/vuejs/core-vapor/pull/5/commits/895f91d2a83c7fdc21e74f6d17f467b62a4b2a78

Reply:
No, wait, 

> Should it be nullable? It could be accessed by setup function.

Which are you referring to? The block?

I was under the impression that I should set the block to the instance during mount, and then remove it upon unmount. Is this correct?

```ts
// mount
instance.container = container
const block = instance.scope.run(() => (instance.block = instance.component()))!
insert(block, instance.container)


// unmount
const { container, block, scope } = instance
scope.stop()
remove(block, container)
```

Reply:
Yes, it's correct. It should be `null` before mounting.